### PR TITLE
Setup project

### PR DESCRIPTION
**Pull request by toliboff**
## Project requirements
*IMPORTANT NOTE*:  This is the first time you will use Gitflow. 
- Create an npm project with webpack.
- Write plain HTML markup with minimum styling (just to make the layout), because you will implement styles in the following steps.
- Don't make API calls (you'll do it in the following lesson).
- Use ES6 modules, with import and export.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+    "html-webpack-plugin": "^5.3.2",
+    "style-loader": "^3.2.1",
+    "webpack-dev-server": "^3.11.2"
+  },
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "watch": "webpack --watch",
+    "start": "webpack serve --open",
+    "build": "webpack"
+  },
```
Comment (vichuge) [inline package.json]: - [ ] Please, add the next function on the script. `"dev": "webpack --mode development",`

File: src/index.html [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+    <section class="add-score">
+      <h2>Add your score</h2>
+
+      <form action="#">
+        <input type="text" id="name" placeholder="Your name" autocomplete="off">
+        <input type="text" id="score" placeholder="Your score" autocomplete="off">
+        <button type="submit" id="submitBtn">Submit</button>
+      </form>
+    </section>
+  </main>
```
Comment (vichuge) [inline src/index.html]: - [ ] It looks like you add everything directly on the `index.html` file. In this case, try to use index.html main contain an only root element (in this case you use `<script src="main.js"></script>`). After you called root, try to add all this on the HTML file using js. Remember to follow all ES6 best practices.

**Review by vichuge — CHANGES_REQUESTED** [review]
# Required Changes ♻️ ⚠️ 🚦 

Hi dear student, 

Good job so far!
You still need to work on some issues to go to the next project, but you are almost there!

mario_ladders

## Good points 👍 
You used proper GitHub Flow ✔️
Your linter checks are all passing ✔️
Your README and PR message are descriptive ✔️

## Aspects to improve ♻️

- [ ] Check the comments under the review.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 3]
```diff
@@ -1 +1,49 @@
...
+:bust_in_silhouette: Tolibjon Tolibov
+- GitHub: [@toliboff](https://https://github.com/toliboff)
+- Twitter: [@tolib_tolibov](https://twitter.com/tolib_tolibov)
+- LinkedIn: [Tolibjon](https://linkedin.com/in/tolibjon-tolibov)
+
+# Contributing
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page.](https://github.com/toliboff//issues)
+
```
Comment (carlos-ssh) [inline README.md:40]: [OPTIONAL]
- Provide a License.

**Review by carlos-ssh — APPROVED** [review]
Hi @toliboff, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**